.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Report reproducible software issues so we can improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**System information**
+ - Cantera version: [e.g. 2.4]
```

Comment:
Add a blank line above the list

Reply:
MD formatting was adopted from templates GitHub suggested. Yes, they're not compliant! (and I obviously didn't check)